File: CS/IssuesContext/Issue.cs
using System;

namespace InfiniteAsyncSourceEFSample {
    public class Issue {
        public int Id { get; set; }
        public string Subject { get; set; }
        public DateTime Created { get; set; }
        public int Votes { get; set; }
        public Priority Priority { get; set; }
    }
    public enum Priority { Low, BelowNormal, Normal, AboveNormal, High }
}

File: VB/IssuesContext/Issue.vb
Imports System

Namespace InfiniteAsyncSourceEFSample
	Public Class Issue
		Public Property Id() As Integer
		Public Property Subject() As String
		Public Property Created() As Date
		Public Property Votes() As Integer
		Public Property Priority() As Priority
	End Class
	Public Enum Priority
		Low
		BelowNormal
		Normal
		AboveNormal
		High
	End Enum
End Namespace
